htdocs/takepos/ajax/ajax.php

```diff
@@ -283,6 +289,17 @@
 		$sql .= $hookmanager->resPrint;
 	}
 
+	if (getDolGlobalInt('TAKEPOS_PRODUCT_IN_STOCK') == 1 && empty(getDolGlobalInt('CASHDESK_ID_WAREHOUSE'.$_SESSION['takeposterminal']))) {
```

Comment:
empty is not allowed and useless with getDolGlobalInt

Reply:
@eldy, ok i've fixed it